test/permission.js

```diff
@@ -0,0 +1,41 @@
+var _Promise = require('bluebird');
+
+var chai = require('chai');
+var sinon = require('sinon');
+
+var errors = require('../api/v1/errors');
+var utils = require('../api/v1/utils');
+var PermissionService = require('../api/v1/services/PermissionService.js');
+var User = require('../api/v1/models/User.js');
+
+var assert = chai.assert;
+var expect = chai.expect;
+
+describe('PermissionService', function(){
+    describe('canCreateUser', function(){
+        it('allows creation by SUPERUSER', function(done){
+            var testUser = User.forge({ id: 1, email: 'new@example.com' });
+            testUser.setPassword('password123').then(function () {
+                testUser.related('roles').add({ role: utils.roles.SUPERUSER});
+                var allow = PermissionService.canCreateUser(testUser);
+                expect(allow).to.eventually.equal(true).and.notify(done);
+           });
+        });
+        it('allows creation of COMMON by ORGANIZER', function(done){
+            var testUser = User.forge({ id: 1, email: 'new@example.com' });
```

Comment:
Try to refactor this test. It looks like it was almost copied and pasted from the test above. It'd be better to make a helper function that returns the texting context `allow` for you given some input parameters.